docs/it/docs/index.md

```diff
@@ -1,12 +1,8 @@
-
-{!../../../docs/missing-translation.md!}
-
-
 <p align="center">
   <a href="https://fastapi.tiangolo.com"><img src="https://fastapi.tiangolo.com/img/logo-margin/logo-teal.png" alt="FastAPI"></a>
 </p>
 <p align="center">
-    <em>FastAPI framework, high performance, easy to learn, fast to code, ready for production</em>
+    <em>Framework FastAPI, alte prestazioni, facile da imparare, veloce da programmare, pronto per il rilascio in produzione</em>
```

Comment:
<em>Framework FastAPI, prestazioni elevate, facile da imparare, veloce da programmare, pronto per il rilascio in produzione</em>